engine/tests/phpunit/Elgg/Notifications/NotificationsServiceTest.php

```diff
@@ -12,14 +12,32 @@ public function setUp() {
 			->disableOriginalConstructor()
 			->getMock();
 		$this->sub = new \Elgg\Notifications\SubscriptionsService($dbMock);
+		$this->translator = new \Elgg\I18n\Translator();
 		$this->session = \ElggSession::getMock();
 
+		// User mock that supports calling $user->isBanned()
+		$user_123 = $this->getMockBuilder('\ElggEntity')
+			->disableOriginalConstructor()
+			->getMock();
+		$user_123->expects($this->any())
+			->method('isBanned')
+			->will($this->returnValue(false));
```

Comment:
For some reason I'm getting the error:

> Call to undefined method Mock_ElggEntity_0b935453::isBanned()

Am I defining the mock method incorrectly?